src/components/forms/DataTypes.ts

```diff
@@ -0,0 +1,48 @@
+import {SelectOptionWithKeyNameAndAbbreviation} from './SelectOptionInterface';
+
+
+export class DataTypes {
+    static CITE: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'CITE',
+        name: 'CITE-seq'
+    };
+    static HYB: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'HYB',
+        name: 'Hybrid Capture'
+    };
+    static RNA: SelectOptionWithKeyNameAndAbbreviation = {
+        key:'RNA',
+        name: 'RNA-Seq'
+    }
+    static SCRNA: SelectOptionWithKeyNameAndAbbreviation = {
+        key:'scRNA',
+        name: 'scRNA-Seq'
+    }
+    static SPT: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'SPT',
+        name: 'Spatial Transcriptomics'
+    }
+    static SNRNA: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'snRNA',
+        name: 'snRNA-Seq'
+    }
+    static WGS: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'WGS',
+        abbreviation:'WGS',
+        name: 'Whole Genome'
+    }
+    static WES: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'WES',
+        abbreviation: 'WES',
+        name: 'Whole Exome'
+    }
+
+    static dataTypesList = [DataTypes.CITE,
+    DataTypes.HYB,
+    DataTypes.RNA,
+    DataTypes.SCRNA,
+    DataTypes.SPT,
+    DataTypes.SNRNA,
+    DataTypes.WGS,
+    DataTypes.WES]
```

Comment:
Should this be pulled into a separate type? (Ditto for the other lists)

Reply:
Good question.  I may iterate on this and eventually pull it into a type, but didn't want to go there until I needed to do that.  The IDE correctly infers dataTypesList as an array of SelectOptionWithKeyNameAndAbbreviation. 
One of the things I'm thinking about is if these even stay in the client since they're shared between the front end for selection and backend for conformance checking.  In some ways, I really like how jade-data-repo leverages OpenAPI for sharing this information between various consumers.  I might be more inclined to go that route in the future with this - but it warrants some discussion.
I was starting to see where the larger form I was working on was getting polluted by long static lists and took a stab at minimizing those to get started.

Reply:
I've added type declarations to each of the arrays so it's explicit for now.